osgp/protocol-adapter-dlms/osgp-dlms/src/main/java/org/opensmartgridplatform/dlms/interfaceclass/attribute/ProfileGenericAttribute.java

```diff
@@ -6,25 +6,36 @@
 
 package org.opensmartgridplatform.dlms.interfaceclass.attribute;
 
+import static org.opensmartgridplatform.dlms.interfaceclass.attribute.AttributeType.CAPTURE_OBJECT_DEFINITION;
+import static org.opensmartgridplatform.dlms.interfaceclass.attribute.AttributeType.CAPTURE_OBJECT_LIST;
+
 import org.opensmartgridplatform.dlms.interfaceclass.InterfaceClass;
 
 /** This class contains the attributes defined for IC ProfileGeneric. */
 public enum ProfileGenericAttribute implements AttributeClass {
   LOGICAL_NAME(1),
   BUFFER(2),
-  CAPTURE_OBJECTS(3),
+  CAPTURE_OBJECTS(3, CAPTURE_OBJECT_LIST),
   CAPTURE_PERIOD(4),
-  SORT_METHOD(5),
-  SORT_OBJECT(6),
+  SORT_METHOD(5, org.opensmartgridplatform.dlms.interfaceclass.attribute.AttributeType.SORT_METHOD),
```

Comment:
A static import won't work, because `SORT_METHOD` in AttributeType clashes with the attribute name `SORT_METHOD`. Instead, I have shortened it to `AttributeType.SORT_METHOD`.